### PR TITLE
Update Using Soto with Vapor part of README

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,14 +168,12 @@ final class MyController {
...
-                let message = SES.Message(body:SES.Body(text:SES.Content(data:emailData.message)), subject:SES.Content(data:emailData.subject))
-                let sendEmailRequest = SES.SendEmailRequest(destination: destination, message: message, source:"awssdkswift@me.com")
+        let emailData = try req.content.decode(EmailData.self)
+        let destination = SES.Destination(toAddresses: [emailData.address])
+        let message = SES.Message(body:SES.Body(text:SES.Content(data:emailData.message)), subject:SES.Content(data:emailData.subject))
+        let sendEmailRequest = SES.SendEmailRequest(destination: destination, message: message, source:"awssdkswift@me.com")
 
-                return client.sendEmail(sendEmailRequest)
-            }
+        return client.sendEmail(sendEmailRequest)
```
Comment (0xTim) [inline README.md:176]: would it be better to use `req.client` here to avoid the hop?

File: README.md [hunk 2]
```diff
@@ -168,14 +168,12 @@ final class MyController {
...
             .hopTo(eventLoop: req.eventLoop)
             .map { response -> HTTPResponseStatus in
```
Comment (0xTim) [inline README.md:178]: Also, `.transform(to: .ok)` is more 'Vapor-y'

File: README.md (hunk 1 above)
Comment (adam-fowler) [reply README.md:176]: You are thinking about soto v5 here surely

File: README.md (hunk 2 above)
Comment (adam-fowler) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (0xTim) [reply README.md:176]: Ah ok I see. Yeah ok leave it as is. I guess the Vapor way would be to do `req.ses.sendEmail` but fitting that into the docs is probably out of scope

Comment (adam-fowler) [reply README.md:176]: Yeah the V5 docs cover this in a lot more detail, but for V4 I'll stick to this

**Review by 0xTim — APPROVED** [review]
LGTM

**Pull request merged**